app/shared/schemas.py

```diff
@@ -38,6 +38,7 @@ class TaskResult(Task):
 
 
 class DeleteResponse(Task):
+    id: str
     deleted: bool
```

Comment:
For `DeleteResponse` to mimic the `JSONResponse` correctly, I had to add a field.

Reply:
in this case it should not depend on Task then, as that has its own id property

Reply:
Let me see if that works out.

Reply:
Done!